Sources/MapboxDirections/MapMatching/MapMatchingResponse.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+class MapMatchingResponse: Decodable {
```

Comment:
This name is confusing: MapMatchingResponse and MatchResponse both decode Map Matching API responses, but MapMatchingResponse produces Routes while MatchResponse produces Matches. Each API should have only one response decoder, and it should be up to the `calculate(_:completionHandler:)` method or some helper method to convert Matches to Routes.

The next step after that would be to make MatchResponse and RouteResponse public and return those objects in the completion handlers. That would give us a place to put stuff like response timestamps and UUIDs instead of duplicating them on each individual result.

Reply:
→ #391